compiler/noirc_frontend/src/tests/traits.rs

```diff
@@ -263,6 +263,56 @@ fn errors_if_impl_trait_constraint_is_not_satisfied() {
 }
 
 #[test]
+fn regression_6314_single() {
```

Comment:
Can you rename these so that it's clear what it's testing without having to refer to GitHub?

Reply:
Best to have a look to the issue in a comment instead.